zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Approve.java

```diff
@@ -76,13 +79,24 @@ public boolean approveCollection(HttpServletRequest request, HttpServletResponse
         return true;
     }
 
-    private long getOverrideKey(HttpServletRequest request) {
-        String overrideKey = request.getParameter("overrideKey");
-        long key = 0;
+    /**
+     * Get the "overrideKey" request parameter as a {@link Long}.
+     *
+     * @param request the {@link HttpServletRequest} to get parameter from.
+     * @return the value as a {@link Long} if present otherwise return null if value is null/empty or an invalid
+     * numeric value.
+     */
+    Long getOverrideKey(HttpServletRequest request) {
+        String overrideKey = request.getParameter(OVERRIDE_KEY_PARAM);
+        if (StringUtils.isEmpty(overrideKey)) {
+            return null;
+        }
+
+        Long key = null;
         try {
             key = Long.valueOf(overrideKey);
         } catch (NumberFormatException ex) {
-            // use default
+            info().exception(ex).log("failed to convert override parameter to long");
```

Comment:
should this be error() ?

Reply:
No. In this case it would continue with null. I added the log just for visibility